core/Node.js

```diff
@@ -1396,7 +1396,11 @@ Node.prototype.update = function update (time){
 
     while (queue.length) {
         item = this._components[queue.shift()];
-        if (item && item.onUpdate) item.onUpdate(time);
+        if (item && item.onUpdate) {
+            item.onUpdate(time);
+
+            if (item.onUpdated) item.onUpdated(time);
```

Comment:
What is, onUpdated? Given that it is called two lines after onUpdate with the same time argument, what is its purpose?